index.html

```diff
@@ -147,9 +155,13 @@ <h2 class="card-name-2">Skills</h2>
     </div>
     </ul>
     </section>
+<div class="contact-container-2">
+  <div class="contact-container">
+    <div class="para-form-container">
+    <div class="para-form">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div> 
```

Comment:
While you are fixing the requested changes, unfortunately, you broke the previous design. "I'm always interested in hearing about new projects, so if you'd like to chat please get in touch." section is not aligned to the center horizontally anymore. Kindly fix that.
It was like that before:
<img width="1112" alt="Screenshot 2023-04-06 at 12 11 51 AM" src="https://user-images.githubusercontent.com/25887253/230213286-ed3afeb2-567b-4870-aae5-e4e498db0010.png">
After fixing the requested changes from the reviewers, now:
<img width="1112" alt="Screenshot 2023-04-06 at 12 12 08 AM" src="https://user-images.githubusercontent.com/25887253/230213565-4e0adf9d-a824-4f17-acaa-1f6023378797.png">
How it should be:
<img width="606" alt="Screenshot 2023-04-06 at 12 12 37 AM" src="https://user-images.githubusercontent.com/25887253/230213647-62187643-07b7-4037-9028-680dd4caf8b9.png">